internal/configs/configmaps.go

```diff
@@ -499,6 +499,15 @@ func ParseConfigMap(cfgm *v1.ConfigMap, nginxPlus bool, hasAppProtect bool, hasA
 				glog.Error("ConfigMap Key 'app-protect-physical-memory-thresholds' must follow pattern: 'high=<0 - 100> low=<0 - 100>'. Ignoring.")
 			}
 		}
+		if appProtectReconnectPeriod, exists := cfgm.Data["app-protect-reconnect-period-seconds"]; exists {
+			if period, err := ParseFloat64(appProtectReconnectPeriod); err == nil {
+				if period > 0 && period <= 60 {
+					cfgParams.MainAppProtectReconnectPeriod = appProtectReconnectPeriod
+				}
+			} else {
+				glog.Error("ConfigMap Key 'app-protect-reconnect-period-second' must have value between '0' and '60'. '0' is illegal. Ignoring.")
+			}
+		}
 	}
```

Comment:
@lucacome Thx for review. Here is suggestion to validate period. Please let me know if its' ugly or I misunderstood smth.
```suggestion
		if appProtectReconnectPeriod, exists, err := ParseFloat64(cfgm.Data["app-protect-reconnect-period-seconds"]); exists {
			if err != nil || appProtectReconnectPeriod <= 0 || appProtectReconnectPeriod > 60 {
				glog.Error("ConfigMap Key 'app-protect-reconnect-period-second' must have value between '0' and '60'. '0' is illegal. Ignoring.")
			} else {
				cfgParams.MainAppProtectReconnectPeriod = appProtectReconnectPeriod
			}
		}
	}
```

Reply:
Unfortunately that doesn't work, I would do something lilke
```suggestion
		if appProtectReconnectPeriod, exists := cfgm.Data["app-protect-reconnect-period-seconds"]; exists {
			period, err := ParseFloat64(appProtectReconnectPeriod)
			if err == nil && period > 0 && period <= 60 {
				cfgParams.MainAppProtectReconnectPeriod = appProtectReconnectPeriod
			} else {
				glog.Error("ConfigMap Key 'app-protect-reconnect-period-second' must have value between '0' and '60'. '0' is illegal. Ignoring.")
			}
		}
	}
```

Reply:
yeah, you right. Thank you!